app/controllers/corporate_controller.rb

```diff
@@ -18,7 +18,8 @@ def show
   end
 
   def create
-    @partner = CorporatePartner.new(partner_params)
+    @partner = CorporatePartner.find_or_initialize_by(params[:email])
```

Comment:
Oopsie, good call on that. Resolved here https://github.com/moneyadviceservice/frontend/commit/101682bc11ebb6bd495cfeeb42c8abc6f96198c9. So `find_or_initialize_by` does expect a hash of attributes but it still somehow worked when passed a string..weird